package.json

```diff
@@ -6,14 +6,13 @@
     "test-py": "python3 -m unittest discover tests",
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle static/css/modules static/js/modules",
     "watch": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
-    "build": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace 'static/css/**/*.css' && postcss --use cssnano --dir static/minified 'static/css/**/*.css' && yarn run copy-3rd-party-js",
+    "build": "node-sass --include-path node_modules static/sass --output-style compressed --output static/css && postcss --use autoprefixer --replace 'static/css/**/*.css' --dir static/minified 'static/css/**/*.css' && yarn run copy-3rd-party-js",
```

Comment:
Thanks, you can make much more savings by removing the inline source maps.

I recommend:
```
    "build": "node-sass --include-path node_modules static/sass --source-map true --output-style compressed --output static/css && postcss --map false --use autoprefixer --replace 'static/css/**/*.css' && yarn run copy-3rd-party-js",
```

Reply:
Ok, done

Reply:
I'm curious - are you saying that without `--source-map true`, `node-sass` will put source maps inline? And with it, it creates them as a separate file? That's odd default behaviour.

Reply:
That is right, the default map option is `--map inline`.

Reply:
@nottrobin https://github.com/postcss/postcss/blob/master/docs/source-maps.md#postcss-and-source-maps